src/CbcCInterface.jl

```diff
@@ -214,21 +214,25 @@ end
 @getproperty Float64 getObjSense
 
 for s in (:getRowLower, :getRowUpper, :getRowActivity)
-    @eval function ($s)(prob::CbcModel)
+    unsafe_s = Symbol("unsafe", '_', s)
+    @eval function ($unsafe_s)(prob::CbcModel)
         check_problem(prob)
         nrow = Int(getNumRows(prob))
         p = @cbc_ccall $s Ptr{Float64} (Ptr{Cvoid},) prob
-        return copy(unsafe_wrap(Array,p,(nrow,)))
+        return unsafe_wrap(Array,p,(nrow,))
```

Comment:
Nit: spaces after comments to follow the JuMP style guide: `unsafe_wrap(Array, p, (nrow,))`. Yes, this was already there but when touching a line you should generally fix style issue.